### DISCUSSION TITLE
python version error

### DISCUSSION THREAD

**Discussion by knrdl** [aquasecurity/trivy#4355] [Q&A] [answered]
### Description

```
INFO    Detecting pip vulnerabilities...
WARN    python version error (0.27.*): malformed version: 0.27.*
```

### Desired Behavior

`0.27.*` is a valid package version notation in requirements.txt:
```
asyncpg==0.27.*
```

### Actual Behavior

trivy cannot parse the notation

### Reproduction Steps

```bash
1. Write a requirements.txt file with `asyncpg==0.27.*`
2. Scan it with trivy
```

### Target

Filesystem

### Scanner

Vulnerability

### Output Format

Table

### Mode

Standalone

### Debug Output

```bash
DEBUG   No secret config detected: trivy-secret.yaml
DEBUG   Walk the file tree rooted at '/tmp/fanal-remote150471596' in parallel
DEBUG   OS is not detected.
INFO    Number of language-specific files: 1
INFO    Detecting pip vulnerabilities...
DEBUG   Detecting library vulnerabilities, type: pip, path: app/requirements.txt
WARN    python version error (0.27.*): malformed version: 0.27.*
```

### Operating System

linux/docker

### Version

```bash
Version: 0.41.0
Vulnerability DB:
  Version: 2
  UpdatedAt: 2023-05-13 18:07:06.906848964 +0000 UTC
  NextUpdate: 2023-05-14 00:07:06.906848564 +0000 UTC
  DownloadedAt: 2023-05-13 20:41:55.265343851 +0000 UTC
Policy Bundle:
  Digest: sha256:97988c6fc189faa28aa7fc2fc93fb2605c46d2f6ae4092a70e03697ba1d6e620
  DownloadedAt: 2023-05-13 20:41:56.449408525 +0000 UTC
```

### Checklist

- [ ] Run `trivy --reset`
- [ ] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by knqyf263** [comment, marked as answer]
As documented here, `*` is not supported now. If you want to request the enhancement, you can open a new discussion.
https://aquasecurity.github.io/trivy/v0.41/docs/scanner/vulnerability/language/python/
